pkg/cluster/cluster.go

```diff
@@ -26,16 +26,26 @@ func Start(ctx context.Context, slug string, dataDir string) error {
 	}
 
 	// start the scheduler on port 11251 (this is a non-standard port)
-	// wg.Add(1)
-	// if err := runScheduler(ctx, &wg, dataDir); err != nil {
+	// if err := runScheduler(ctx, dataDir); err != nil {
 	// 	return errors.Wrap(err, "start scheduler")
 	// }
 
-	// // start the controller manager on port 11252 (non standard)
-	// wg.Add(1)
-	// if err := runController(ctx, &wg, dataDir); err != nil {
+	// start the controller manager on port 11252 (non standard)
+	// TODO the controller should start
+	// if err := runController(ctx, dataDir); err != nil {
```

Comment:
Disabling controllermanager for now since this is not starting. This needs to be figured out.